manifest.toml

```diff
@@ -39,16 +39,25 @@ ram.runtime = "200M"
     type = "group"
     default = "admins"
 
-    [install.admin]
-    type = "user"
+[resources]
+    [resources.sources.pgadmin_prebuilt_armv7_bookworm]
+    prefetch = false
+    armhf.url = "https://github.com/YunoHost-Apps/pgadmin_python_build/releases/download/v4-8.3/pgadmin_4-8.3-bookworm-bin1_armv7l.tar.gz"
+    armhf.sha256 = "b2fdf884fe74e2e6a41d8f5f8f4b1967bc4ad6fffcfa5a76fc34ef6fe6c62e7e"
```

Comment:
Why using, maintaining etc this prebuilt artifact for armhf ?

Reply:
Because it about 3 hours to build !!!! And we also need rust and many others things to build as no build wheel are available...

Reply:
You have more info about wheel here: https://realpython.com/python-wheels/#linux-building-manylinux-wheels

Shortly installing python package on x86-64, i686, aarch64 are generally easy and short as binary are available. For others architecture it could be sometime more complicated or even very complicated depending of the lib that you would like to install. By example the `cryptography` and `lxml` lib have many dependences to build and take a long time to build especially on arm.

So the other solution would be to just drop support for all arch on manifest and to support only architecture which has prebuild wheel.